packages/workbox-build/src/templates/sw-template.js

```diff
@@ -14,23 +14,41 @@
   limitations under the License.
 */
 
-module.exports = `<% if (importScripts) { %>
+module.exports = `/**
+ * Welcome to your Workbox-powered service worker!
+ *
+ * Here are some next steps:
+ *
+ * - Your web app needs to register this file.
+ *   See https://goo.gl/DNGzMp
+ *
+ * - Disable HTTP caching for this file.
+ *   See https://goo.gl/rWuKgq
+ *
+ * The rest of the code in this file is auto-generated. Please don't update this
+ * code directly; instead, make changes to your Workbox build configuration
+ * and re-run your build process to regenerate this file with updated code.
+ * See https://goo.gl/YYPcyY
+ */
+ 
+<% if (importScripts) { %>
 importScripts(<%= importScripts.map(JSON.stringify).join(',') %>);
 <% } %>
+
+<% if (modulePathPrefix) { %>workbox.setConfig({modulePathPrefix: <%= JSON.stringify(modulePathPrefix) %>});<% } %>
+<% if (cacheId) { %>workbox.core.setCacheNameDetails({prefix: <%= JSON.stringify(cacheId) %>});<% } %>
+
+<% if (skipWaiting) { %>workbox.skipWaiting();<% } %>
+<% if (clientsClaim) { %>workbox.clientsClaim();<% } %>
+
+<% if (Array.isArray(manifestEntries)) {%>
 /**
- * This service worker file needs to be registered by your web application.
- * Please see https://developers.google.com/web/fundamentals/primers/service-workers/registration
- */
-<% if(manifestEntries) {%>
-/**
- * DO NOT EDIT __precacheManifest MANUALLY!
- *
- * The workboxSW.precache() method does the following:
+ * The workboxSW.precacheAndRoute() method does the following:
```

Comment:
In Workbox v3 branch there is a pretty thorough explanation of what precaching is and how it works. Could we shorten this comment to:

"// Efficiently cache and server requests
 // Learn more https://developers.google.com/web/tools/Workbox/modules/workbox-precaching"